cmake/wsrep.cmake

```diff
@@ -16,29 +16,12 @@
 # We need to generate a proper spec file even without --with-wsrep flag,
 # so WSREP_VERSION is produced regardless
 
-<<<<<<< HEAD
 # Set the patch version
-SET(WSREP_PATCH_VERSION "55")
+SET(WSREP_PATCH_VERSION "57")
 # PXC needs this because it still support execvpe approach for forking
 # while initiating SST.
 INCLUDE(CheckFunctionExists)
 CHECK_FUNCTION_EXISTS(execvpe HAVE_EXECVPE)
-||||||| merged common ancestors
-# Set the patch version
-SET(WSREP_PATCH_VERSION "27")
-=======
-OPTION(WITH_WSREP "WSREP replication API (to use, e.g. Galera Replication library)" OFF)
-
-IF (NOT WITH_WSREP)
-  RETURN()
-ENDIF()
-
-FILE(READ "${CMAKE_SOURCE_DIR}/WSREP_VERSION" wVER)
-STRING(REGEX MATCH "WSREP_VERSION_API=([0-9]*)" _ ${wVER})
-SET(WSREP_API_VERSION ${CMAKE_MATCH_1})
-STRING(REGEX MATCH "WSREP_VERSION_PATCH=([0-9]*)" _ ${wVER})
-SET(WSREP_PATCH_VERSION ${CMAKE_MATCH_1})
->>>>>>> wsrep_5.7.36-25.28
```

Comment:
I think it is better we retain upstream's logic of fetching the `WSREP_PATCH_VERSION` from the WSREP_VERSION file. Could you please revert these changes?